class Solution {
    public int reverseDegree(String s) {
        int n= s.length();
        int sum=0;
        for(int i=0;i<n;i++){
            int reversed_val= 26-(s.charAt(i)-'a');
            int result= reversed_val*(i+1);
            sum+=result;
        }
        return sum;
    }
}